index.html

```diff
@@ -41,6 +42,148 @@ <h3 class="connect">LET’S CONNECT</h3>
+                           <img src="assets/icons/Counter.png" alt="counter">
+                           <h4 class="lang-year">Back End Dev</h4>
+                           <img src="assets/icons/Counter.png" alt="counter">
+                           <h4 class="lang-year">2015</h4>
+                        </div>
+                        </div>
+                    <p class="detail-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="btns">
+                        <li class="btn">HTML</li>
+                        <li class="btn">CSS</li>
+                        <li class="btn">JavaScript</li>
+                    </ul>
+                    <button class="detail-btn">See project</button>
+                </div>
+            </div>
+        </div>
+            </div>
+        </section>
+        <section class="about-myself">
+                <div class="myself">
+                    <h2 class="about-me-title">About Myself</h2>
+                    <p class="about-details">Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+                    <h3 class="connect">LET’S CONNECT</h3>
+                    <div class="social-icons">
+                        <ul>
+                            <li><a href=""><img src="assets/icons/github-icon.svg" alt="github" class="img-tag"></a></li>
+                            <li><a href=""><img src="assets/icons/linkedin-icon.svg" alt="linkedin" class="img-tag"></a></li>
+                            <li><a href=""><img src="assets/icons/m-icon.svg" alt="m-icon" class="img-tag"></a></li>
+                            <li><a href=""><img src="assets/icons/twitter-icon.svg" alt="twitter" class="img-tag"></a></li>
+                            <li><a href=""><img src="assets/icons/Vector.svg" alt="vector" class="img-tag"></a></li>
+                        </ul>
+                    </div>
+                    <button class="detail-btn">Get my resume</button>
+                </div>
+        </section>
+
+        <article class="right-content">
+            <h2>Languages</h2>
+            <ul class="languages">
+                <li>
+                    <img src="assets/images/js.png" alt="">
+                    <h4>Javascript</h4>
+                </li>
+                <li>
+                    <img src="assets/images/html.png" alt="">
+                    <h4>HTML</h4>
+                </li>
+                <li>
+                    <img src="assets/images/css.png" alt="">
+                    <h4>CSS</h4>
+                </li>
+            </ul>
+
+            <ul class="tools-container">
+                <li class="tools">framworks</li>
+                <li class="tools">skills</li>
+            </ul>
+        </article>
```

Comment:
- [ ] Good job with this attempt :clap: If you look closely, you will observe that these parts form part of the `About me` section where you show off your technical skill. Kindly nest it in within the `<section class="about-myself"></section>` tag. This makes your code tidier, and easily accessible to style, or in terms of accessibility to users with screen readers. :wink: